.gitignore

```diff
@@ -16,7 +16,6 @@
 
 # Build/Maven directory
 target/
-target/
```

Comment:
Just to be clear, can you clarify why we are deleting this `target` entry from here ?

Reply:
It's a duplicate line

Reply:
oops! :D